Ginger/GingerCoreCommon/VariablesLib/VariableDateTime.cs

```diff
@@ -213,13 +211,21 @@ public string ConvertDateTimeToSpecificFormat(string format)
         {
             return Convert.ToDateTime(this.mInitialDateTime).ToString(format, System.Globalization.CultureInfo.InvariantCulture);
         }
+        public string ConvertMindateTime(string format)
```

Comment:
ConvertDateTimeToSpecificFormat, ConvertMindateTime and ConvertMaxDateTimeToSpecificFormat,  All this 3 functions have same functionlaity. Please combine this 3 functions to 1 function to avoid duplication\repeatation of code